charmcraft.yaml

```diff
@@ -1,11 +1,46 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+name: nfs-server-proxy
+
 type: charm
+
+title: NFS Server Proxy
+summary: Charmed NFS server for non-charmed NFS storage appliances.
+description: |
+  Charmed NFS server for non-charmed NFS storage appliances.
+
+  Enable NFS Client charms to mount non-charmed NFS shares.
+links:
+  source: 
+  - https://github.com/canonical/nfs-server-proxy-operator
+  issues: 
+  - https://github.com/canonical/nfs-server-proxy-operator/issues
+
+peers:
+  server-peers:
+    interface: server_peers
```

Comment:
Will the assumption of a peer relation existing in `FilesystemProvider` also work for CephFS + microceph?

Reply:
CephFS doesn't use the library, it just sets the databag directly, so it doesn't really matter for that case.

Reply:
:+1: makes sense, but if a provider charm did want to use this library, they'd likely need to add a peer relation then, correct?

Shouldn't be too much of an issue depending on the scale of the provider.